std/src/main/scala/algebra/std/vector.scala

```diff
@@ -0,0 +1,41 @@
+package algebra
+package std
+
+import scala.annotation.tailrec
+
+package object vector extends VectorInstances
+
+trait VectorInstances {
+  implicit def vectorOrder[A: Order]: Order[Vector[A]] = new VectorOrder[A]
+  implicit def vectorMonoid[A]: Monoid[Vector[A]] = new VectorMonoid[A]
+}
+
+final class VectorOrder[A](implicit ev: Order[A]) extends Order[Vector[A]] {
+  def compare(x: Vector[A], y: Vector[A]): Int = {
+    val lengthX = x.length
+    val lengthY = y.length
+
+    @tailrec
+    def cmp(index: Int): Int = {
+      if (index == lengthX) {
+        lengthX - lengthY
+      } else if (index == lengthY) {
+        1
+      } else {
+        val n = ev.compare(x(index), y(index))
+        if (n == 0) cmp(index + 1) else n
+      }
+    }
+
+    cmp(0)
+  }
+}
+
+class VectorMonoid[A] extends Monoid[Vector[A]] {
```

Comment:
Yes, I overlooked that import vector._ would give you this one. In algebird we made `Monoid[List[T]] ` the free monoid here, but we used `Monoid[T] => Monoid[Vector[T]]` to do pointwise addition (with padding of zeros to the longer vector, but what we really want is the length as a type parameter and only add vectors of the same length, but we didn't geek out with type-level Nats).